docs/BUILD_AND_TEST.md

```diff
@@ -1,57 +1,58 @@
 
-    ```
-    git remote add origin git@github.com:<yourGithubID>/snap.git
-    ```
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--> 
+# Build and Test
+This guide gets you started with building and testing Snap. If you have commits you want to contribute, review the [CONTRIBUTING file](../CONTRIBUTING.md) for a shorter list of what we look for and come back here if you need to verify your environment is configured correctly.
 
-    * To push to your fork later, either do a `git push origin master` or do `git checkout -b <someBranchName>` then you can do `git push`.
+## Getting Started
+If you prefer a video walkthrough for this process, watch this video: https://vimeo.com/161561815
 
-## Build
+To build the Snap Framework you'll need:
+* [Golang >= 1.5](https://golang.org)
+    * Should be [downloaded](https://golang.org/dl/) and [installed](https://golang.org/doc/install)
+* [GNU Make](https://www.gnu.org/software/make/)
+* [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 
-Godeps is a dependency for running the make task(s) required for the build process. If it is not already installed,
-install it into your GOPATH and ensure that its built executables are in your PATH.
+The instructions below assume that the `GOPATH` environment variable has been set properly. Many of us use [go version manager (gvm)](https://github.com/moovweb/gvm) to easily switch between Go versions.
 
+Now you can install Snap into your `GOPATH`: 
 ```
-# first check to see if it is installed
-which godep
-# install it if necessary
-export PATH=$GOPATH/bin/
-go get github.com/tools/godep
+$ go get github.com/intelsdi-x/snap
+$ cd $GOPATH/src/github.com/intelsdi-x/snap
 ```
 
-In the /snap directory there's a `Makefile` that builds all dependencies and snap.
-To get dependencies and build snap just run:  
-
+[Godeps](https://github.com/tools/godep) is a dependency for running the `make` task(s) required for the build process. If it is not already installed, install `godeps` now:
 ```
-make
+$ # first check to see if it is installed
+$ which godep
+$ # if not installed, do so
+$ # then download and set your path
+$ go get github.com/tools/godep
+$ export PATH=$GOPATH/bin/
 ```
 
-It runs the following `make deps` and `make all` commands.
+In the `snap/` directory there's a `Makefile` that builds all dependencies and then the Snap Framework binaries. To get dependencies and build Snap run:  
+```
+$ cd $GOPATH/src/github.com/intelsdi-x/snap
+$ make
+```
 
-Alternatively, you can run `make` with any of the following targets:
+It runs `make deps` and `make all` commands for you. Alternatively, you can run `make` with any of the following parameters:
 
 * `default`: runs make deps and make all
```

Comment:
This is the equivalent to `make` so they don't need to do `make default`. Maybe disclaimer?